<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login1</class>
 <widget class="QMainWindow" name="login1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>使用中。。。</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>☝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f1sure">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>借阅</string>
    </property>
   </widget>
   <widget class="QLabel" name="f1l1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>借阅中。。。</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="f1name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="f1l2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>图书编号：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f1search">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查询书籍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f1delete">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f1sure1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="fmoney">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="f2l1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>已借书籍。。。</string>
    </property>
   </widget>
   <widget class="QTableView" name="f2info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);color: rgb(255, 155, 170);

background-image: url(:/new/prefix2/xingkong2.JPG);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f2xujie">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>申请续借</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f2back">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>申请归还</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f2pay">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>缴 费</string>
    </property>
   </widget>
   <widget class="QLabel" name="stext">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);color: rgb(255, 155, 170);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="f3l2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>姓 名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="f3l4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>结束时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="f3starttime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="f3password">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="f3l6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>学 院：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="f3name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="f3l3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>入学时间：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="f3college">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>计算机学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电子信息学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自动化学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>机电学院</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="f3l5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>登陆密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="f3overtime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="f3l7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>年 级：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="f3use">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="f3l1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(131, 255, 255);</string>
    </property>
    <property name="text">
     <string>学 号：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f3sure">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f3delete">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f3change">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="f3co">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="f3gra">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
